backend/plugins/content_api/src/modules/portal/db/models/Sample.ts

```
@@ -0,0 +1,56 @@
+        { _id },
+        { $set: { ...doc } },
+      );
+    }
+
+    /**
+     * Remove contents
+     */
+    public static async removecontents(contentIds: string[]) {
+      return models.contents.deleteMany({ _id: { $in: contentIds } });
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)